deploy/cr.yaml

```diff
@@ -297,6 +297,7 @@ spec:
     gracePeriod: 30
 #    loadBalancerSourceRanges:
 #      - 10.0.0.0/8
+#    loadBalancerIP: 127.0.0.1
```

Comment:
please have a look at ` replicasServiceType: ClusterIP` it is separate service and we need to have loadBalancerIP for this service too 'replicasloadBalancerIP'